Source/websocket/WebSerializer.h

```diff
@@ -423,5 +430,40 @@ namespace Web {
     private:
         HashType _hash;
     };
+
+    namespace JSONRPC {
+
+        class Body : public JSONBodyType<Core::JSONRPC::Message> {
+        public:
+            Body(Body&&) = delete;
+            Body(const Body&) = delete;
+            Body& operator= (Body&&) = delete;
+            Body& operator= (const Body&) = delete;
+
+            Body() = default;
+            ~Body() override = default;
+
+        public:
+            void Clear() {
```

Comment:
Not too important but I guess this should also be am override

Reply:
Correct will adapt it..